docs/interpreter/jdbc.md

```diff
@@ -723,6 +727,26 @@ Before Adding one of the below dependencies, check the Phoenix version first.
 </table>
 
 [Maven Repository: org.apache.tajo:tajo-jdbc](https://mvnrepository.com/artifact/org.apache.tajo/tajo-jdbc)
+## Object Interpolation
+The JDBC interpreter also supports interpolation of `ZeppelinContext` objects into the paragraph text.
+The following example shows one use of this facility:
+
+####In Scala cell:
+```
+z.put("country_code", "KR")
+    // ...
+```
+
+####In later JDBC cell:
+```
```

Comment:
add `sql` - see [L230](https://github.com/apache/zeppelin/pull/2903/files#diff-12ee03bf05e397a25720e23c90890661R230)

Reply:
Added the 'sql'